src/schemas/22-0994/schema.js

```diff
@@ -0,0 +1,187 @@
+import _ from 'lodash/fp';
+import { countries } from '../../common/constants';
+import definitions from '../../common/definitions';
+import {
+  documentTypes526,
+  pciuCountries,
+  pciuStates
+} from '../../common/constants';
+
+const vetTecPrograms = [
+  "NAME",
+  "NAME",
+  "NAME",
+  "NAME",
+  "NAME"
+];
+
+const highTechnologyEmploymentType = [
+  "Computer Programming",
+  "Data Processing",
+  "Computer Software",
+  "Information Sciences",
+  "Media Application"
+];
+
+const salaries = [
+  "<$20,000",
+  "$20,001 - $35,000",
+  "$35,001 - $50,000",
+  "$50,001 - $75,000",
+  ">$75,000"
+];
+
+const baseAddressDef = {
+  type: 'object',
+  required: ['country', 'city', 'addressLine1'],
+  properties: {
+    country: {
+      type: 'string',
+      enum: pciuCountries,
+      default: 'USA'
+    },
+    addressLine1: {
+      type: 'string',
+      maxLength: 20,
+      pattern: "^([-a-zA-Z0-9'.,&#]([-a-zA-Z0-9'.,&# ])?)+$"
+    },
+    addressLine2: {
+      type: 'string',
+      maxLength: 20,
+      pattern: "^([-a-zA-Z0-9'.,&#]([-a-zA-Z0-9'.,&# ])?)+$"
+    },
+    addressLine3: {
+      type: 'string',
+      maxLength: 20,
+      pattern: "^([-a-zA-Z0-9'.,&#]([-a-zA-Z0-9'.,&# ])?)+$"
+    },
+    city: {
+      type: 'string',
+      maxLength: 30,
+      pattern: "^([-a-zA-Z0-9'.#]([-a-zA-Z0-9'.# ])?)+$"
+    },
+    state: {
+      type: 'string',
+      enum: pciuStates.map(state => state.value),
+      enumNames: pciuStates.map(state => state.label)
+    },
+    zipCode: {
+      type: 'string',
+      pattern: '^\\d{5}(?:([-\\s]?)\\d{4})?$'
+    }
+  }
+};
+
+const schema = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: "APPLICATION FOR VETERAN EMPLOYMENT THROUGH TECHNOLOGY EDUCATION COURSES (VET TEC) HIGH TECHNOLOGY PROGRAM",
+  type: 'object',
+  definitions: {
+    phone: definitions.usaPhone,
+    date: definitions.date,
+    fullName: definitions.fullName,
+    ssn: definitions.ssn,
+    bankAccount: definitions.bankAccount,
+    gender: definitions.gender,
+    email: {
+      type: 'string',
+      minLength: 6,
+      maxLength: 80,
+      pattern:
+        '^[_A-Za-z0-9-]+(\\.[_A-Za-z0-9-]+)*@[A-Za-z0-9-]+(\\.[A-Za-z0-9]+)*(\\.[A-Za-z]{2,})$'
+    },
+    address: baseAddressDef,
+  },
+  properties: {
+    applicantFullName: {
+    $ref: "#/definitions/fullName"
+  },
+    applicantSocialSecurityNumber: {
+    $ref: "#/definitions/ssn"
```

Comment:
Do we know if the applicant full name and social security number are going to be collected by the UI?  I know that the all-claims schema didn't actually include the veteran full name because it was actually added to the form data in `vets-api` from the user record instead of being sent from `vets-website`

If that was the case with the 0994, we need to make sure that the name isn't required.

Reply:
I think the name fields will be there but will be pre-populated with answers. As of right now, what I have done in ```bah-0994``` on vets-website is not populating though.